### ISSUE TITLE
Linters do not use dependencies inside `vendor` folder

### ISSUE THREAD

**Issue by nkryuchkov** [skycoin/skywire-testnet#497]
Vendoring mode is not enabled in the `golangci-lint` config and the `go vet` flags. This sometimes leads to broken `Travis CI` builds when dependencies in `vendor` folder and repository are not the same, because `golangci-lint` and `go vet` do not use the `vendor` folder.

An example of a broken build:
https://travis-ci.com/skycoin/skywire/builds/120084033
This happened in https://github.com/skycoin/skywire/pull/496 on commit https://github.com/skycoin/skywire/pull/496/commits/f477fb1a558e0630b5ed69afdaf63c9a0052e9b4. Here, linters cannot find the changed code in the `vendor` folder and try to download it instead which leads to a broken build.

**Comment by jdknives** [comment]
Solved by #498

**Comment by nkryuchkov** [comment]
Adding `-mod=vendor` flag to `go vet` causes the following error:
```
GO111MODULE=on go vet -mod=vendor -all ./...
go vet: mod flag may be set only once
run "go help vet" for more information
make: *** [lint] Error 2
```
